File: addons/calendar_microsoft/models/calendar_event.py
from odoo import fields, models

class Event(models.Model):
    _name = 'calendar.event'
    _inherit = 'calendar.event'

    ms_organizer_event_id = fields.Char('Microsoft Calendar Organizer event Id')
    ms_universal_event_id = fields.Char('Microsoft Calendar Universal event Id')

File: addons/calendar/static/src/models/calendar_event.js
/** @odoo-module **/

import { attr, clear, Model } from '@mail/model';

import fieldUtils from 'web.field_utils';
import { getLangTimeFormat } from 'web.time';

Model({
    name: 'calendar.event',
    fields: {
        allday: attr(),
        attendee_status: attr(),
        formattedStart: attr({
            compute() {
                if (!this.start) {
                    return clear();
                }
                return moment(fieldUtils.parse.datetime(this.start, false, { isUTC: true })).local().format(getLangTimeFormat());
            },
        }),
        id: attr({
            identifying: true,
        }),
        name: attr(),
        start: attr(),
    },
});
